Doc/conf.py

```diff
@@ -30,20 +31,20 @@
 
 # Skip if downstream redistributors haven't installed them
 try:
-    import notfound.extension
+    import notfound.extension  # NoQA: F401
 except ImportError:
     pass
 else:
     extensions.append('notfound.extension')
 try:
-    import sphinxext.opengraph
+    import sphinxext.opengraph  # NoQA: F401
 except ImportError:
     pass
 else:
     extensions.append('sphinxext.opengraph')
 
 
-doctest_global_setup = '''
+doctest_global_setup = """
```

Comment:
Why are these quote styles changing if you have `quote-style = "single"` in the `.ruff.toml` file?

Reply:
From [Ruff's documentation](https://docs.astral.sh/ruff/settings/#format_quote-style):

> In compliance with [PEP 8](https://peps.python.org/pep-0008/) and [PEP 257](https://peps.python.org/pep-0257/), Ruff prefers double quotes for triple quoted strings and docstrings even when using quote-style = "single".

Ruff could add another setting, but I don't think you should / it is worth it, really.

Reply:
Huh, fair enough

Reply:
Possibly we could get a smaller diff for now by doing `quote-style = "preserve"`, and then we could normalize all quote styles in a followup? It feels kinda silly to me to normalize triple-quotes but not other quotes, tbh. (But I don't feel too strongly.)